Templates/Ros2ProjectTemplate/Template/Gem/enabled_gems.cmake

```diff
@@ -18,7 +18,7 @@ set(ENABLED_GEMS
     ImGui
     LandscapeCanvas
     LyShine
-    PhysX
```

Comment:
yikes, we should eventually update this template to not have an `enabled_gems.cmake` file any more.  All these gems should be [moved to the `project.json` like the default template](https://github.com/o3de/o3de/blob/development/Templates/DefaultProject/Template/project.json) and several other changes like updating the [EngineFinder.cmake](https://github.com/o3de/o3de/blob/development/Templates/DefaultProject/Template/cmake/EngineFinder.cmake) and whatever [`CMakeLists.txt`](https://github.com/o3de/o3de/blob/development/Templates/DefaultProject/Template/CMakeLists.txt) changes are needed.